docs/src/styles.css

```
@@ -0,0 +1,516 @@
+
+body {
+  font-family: "JetBrains Mono", monospace;
+  line-height: 1.35;
+}
+
+/* Keep the Gravity root transparent so the brand background shows through, and
+   restore the JetBrains Mono body font (Gravity sets its own sans on .g-root,
+   which would otherwise win over body). */
+.g-root {
+  --g-color-base-background: var(--bg);
+  background: var(--bg);
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)